istio/init-1/profile.yaml

```diff
@@ -23,18 +23,26 @@ spec:
       - name: istio-ingressgateway
         enabled: true
         k8s:
-          service:
-            ports:
-              - port: 80
-                name: http
-              - port: 443
-                name: https
           overlays:
-            # - kind: Service
-            #   name: public-gateway
-            #   patches:
-            #     - path: spec.loadBalancerIP
-            #       value: "34.89.83.239"
+            - kind: Service
+              name: istio-ingressgateway
+              patches:
+                - path: spec.loadBalancerIP
+                  value: "${PUBLIC_IP}"
+      - name: istio-ingressgateway-internal
+        enabled: true
+        k8s:
+          overlays:
```

Comment:
I wonder whether we could delete this new ingressgateway entry and just add a new `-kind: Service` entry to the above `overlays:` section above with the below config to avoid having another ingress pod?

Reply:
Raised as [issue](https://github.com/MHRA/products/issues/974)